crossbeam-skiplist/src/lib.rs

```diff
@@ -257,3 +259,51 @@ pub mod set;
 #[cfg(feature = "std")]
 #[doc(inline)]
 pub use crate::{map::SkipMap, set::SkipSet};
+
+/// Key equivalence trait.
```

Comment:
It might be better to put these in a separate file.

Reply:
Done!